dist/index.html

```diff
@@ -42,11 +45,62 @@
     <script type="text/javascript" src="itkVtkViewer.js">
     </script>
     <script>
-      var container = document.querySelector('.content');
-      var viewer = itkVtkViewer.processURLParameters(container, { fullscreen: true });
+     async function setupViewerForImJoy(container) {
+      const imjoyRPC = await imjoyLoader.loadImJoyRPC({
+        api_version: '0.2.3'
+      })
+      const api = await imjoyRPC.setupRPC({
+        name: 'itk-vtk-viewer',
+        version: '9.23.x',
+        description: '2D / 3D web image, mesh, and point set viewer using itk.js and vtk.js ',
+        type: 'rpc-window',
```

Comment:
Could you please also add a `docs` key that point to a url (e.g. a web page or a markdown file) with the documentation for the ImJoy api?

Reply:
Sounds good -- I'll add a `docs` page at https://kitware.github.io/itk-vtk-viewer/docs/imjoy.html.